deploy/examples/csi/cephfs/groupsnapshot.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-apiVersion: groupsnapshot.storage.k8s.io/v1alpha1
+apiVersion: groupsnapshot.storage.k8s.io/v1beta1
```

Comment:
Anything we need to worry about with the conversion from alpha to beta? No upgrade path needed, right? Since the group snapshots haven't been fully supported yet anyway.

Reply:
Yes correct,  upstream dropped beta API and no backward compatibility support for it

Reply:
I need to update the existing documents,  will update it as well